src/templates/festivals/nimble/index.html

```diff
@@ -0,0 +1,7 @@
+{% for events in event_category_list %}
```

Comment:
Create a generic template for any kind of festival and not just for Nimble. `extend` from base templates to save time.